docs/element-definitions.md

```diff
@@ -1431,7 +1431,7 @@ Example of an HTTP Transaction.
 
 - `element` - `"httpTransaction"`
 - `attributes`
-    - `authSchemes` ([Array][]) - An array of authentication and authorization schemes that apply to the transaction
+    - `authSchemeRequirements` ([Array][][Auth Scheme Requirement][]) - An array of authentication and authorization scheme requirements out of which atleast one SHOULD apply to the transaction.
```

Comment:
Could we get some justification around this design change? What is wrong with the previous name and value. To me without any context this seems like an unnessecery breaking change.